src/main/java/de/bonndan/nivio/input/LabelToFieldProcessor.java

```diff
@@ -54,20 +58,38 @@ private void setValue(ItemDescription item, String name, String value) {
                 myAccessor.setPropertyValue(name, Arrays.asList(o));
                 return;
             }
+
             if (propertyType != null && propertyType.isAssignableFrom(Set.class)) {
                 String[] o = getParts(value);
                 myAccessor.setPropertyValue(name, Set.of(o));
                 return;
             }
 
+            if (propertyType != null && propertyType.isAssignableFrom(Map.class)) {
+                String[] o = getParts(value);
+                Map propertyValue = (Map) myAccessor.getPropertyValue(name);
+                for (int i = 0; i < o.length; i++) {
+                    if (ItemDescription.LINKS_FIELD.equals(name)) {
+                        try {
+                            propertyValue.put(String.valueOf(i+1), new URL(o[i]));
+                        } catch (MalformedURLException e) {
+                            logger.warn("Failed to parse link " + o[i]);
+                        }
+                    } else {
+                        propertyValue.put(String.valueOf(i+1), o[i]);
+                    }
+                }
+                return;
+            }
+
             myAccessor.setPropertyValue(name, value.trim());
         } catch (NotWritablePropertyException e) {
             logger.warn("Failed to write field '" + name + "' via label");
         }
     }
 
     private static String[] getParts(String value) {
```

Comment:
Imho static, albeit private methods should go to the top of a class, even before public methods. What's your opinion @Matthimatiker?

Reply:
My opinion(!) is: public stuff comes first. You are however free to change it.